tests/conftest.py

```diff
@@ -85,3 +88,40 @@ def election_settings(client: FlaskClient, election_id: str) -> None:
     }
     rv = put_json(client, f"/election/{election_id}/settings", settings)
     assert json.loads(rv.data) == {"status": "ok"}
+
+
+@pytest.fixture
+def manifests(client: FlaskClient, election_id: str, jurisdiction_ids: List[str]):
+    rv = client.put(
+        f"/election/{election_id}/jurisdiction/{jurisdiction_ids[0]}/manifest",
+        data={
+            "manifest": (
+                io.BytesIO(
+                    b"Batch Name,Number of Ballots\n"
+                    b"1,23\n"
+                    b"2,101\n"
+                    b"3,122\n"
+                    b"4,400"
+                ),
+                "manifest.csv",
+            )
+        },
+    )
+    assert rv.status_code == 200
```

Comment:
One thing I've started doing with these is to [add a message that includes the response](https://github.com/votingworks/arlo/blob/6fdab2ff06d8cc08739e2eaf4a3f1016a3c81566/tests/routes_tests/test_election_settings.py#L34). Otherwise, it can become difficult to determine why this is happening with a `200 != 400` message.

Reply:
I guess that's a good argument for the pattern that existing tests were using: `assert json.loads(rv.data) == {"status": "ok"}`. I started checking status code instead because it seemed simpler, but maybe I'll just go back. Or make an `assert_ok(rv)` helper that does all the things...